CIPs/0007.md

```diff
@@ -0,0 +1,65 @@
+# CIP [0007]: Affordable Gas Downtime Slasher
+
+- Date: 2020-05-05
+- Author: @gastonponti (with a lot of feedback and ideas from @m-chrzan)
+- Status: Implemented
+
+## Overview
+
+This proposal includes improvements to Celo's downtime slashing, such that it can be evaluated within the block gas limit.
+
+In order to promote a stable network, the Celo protocol includes slashing for downtime such that if a validator fails to participate in consensus for a given amount of time, they may be slashed. More information about slashing in the Celo protocol can be found on [docs.celo.org](https://docs.celo.org/celo-codebase/protocol/proof-of-stake/penalties)
+
+Currently, slashable downtime is defined as a window of blocks, greater than a predefined length, in which no block's parent aggregated seal includes a signature from the validator in question. It's currently implementation checks and executes slashing in a single transaction which requires an amount of gas linear in the required window size.
```

Comment:
The definition did not change. Removing the work "currently" would help clarify that.
```suggestion
Slashable downtime is defined as a window of blocks, greater than a predefined length, in which no block's parent aggregated seal includes a signature from the validator in question. It's currently implementation checks and executes slashing in a single transaction which requires an amount of gas linear in the required window size.
```